interface-definitions/dhcp-server.xml.in

```diff
@@ -142,6 +142,11 @@
                 boot file is to be loaded</help>
                     </properties>
                   </leafNode>
+		  <leafNode name="bootfile-size">
+		    <properties>
+		      <help>Bootstrape file size in 512 byte blocks</help>
```

Comment:
Typo! Also, I have to admit that I haven't used that option, I only used the boot-file one and TFTP clients pickes it up just fine. I wonder why it's in 512 byte blocks, and can the CLI allow the user to specify it in k/mbytes instead and convert it to blocks internally?

Reply:
Sorry about that - repaired the typo. Whilst I assume most users using this relatively obscure option have the right number handy already, a conversion feature would indeed be handy - but I unfortunately don't know how to implement that.

Reply:
I think it's best to add a fixup in the script that is calling that template: https://github.com/vyos/vyos-1x/blob/current/src/conf_mode/dhcp_server.py#L281

The `generate()` function takes the config dict as an argument, so you can check for that option there and modify the value.

Reply:
Replied in main thread (https://github.com/vyos/vyos-1x/pull/1148#issuecomment-1027297794).